src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -120,6 +120,12 @@ const createUiSchemaForObject = (ionFormField, remediationForm, transformedResp,
     // 2. when `select-authenticator-enroll`,
     // create customize type so that display authenticator list as selectable list
     Object.assign(uiSchema, getAuthenticatorsEnrollUiSchema(ionFormField));
+  } else if (ionFormField.name === 'scopes')  {
+    const options = ionFormField.options.map(({label, desc}) => {
+      return {name: label, displayName: label, description: desc};
+    });
+    // setting 'type' here to add a specific View in FormInputFactory.create
+    Object.assign(uiSchema, {options, type: 'scopes'});
```

Comment:
- Be more explicit like `oauth2-admin-scopes` ?
- wonder if the remediation form name could be more explicit like `admin-scopes` ? (question to api team)

Reply:
The remediation form name `admin-consent` seems more appropriate since the view/flow is referred to as Consent, where scopes is a property of Consent.
I can change the `type`.

Reply:
I changed type to `adminConsentScopes`